<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectConfiguration</class>
 <widget class="QDialog" name="SelectConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_DBNameLabel">
         <property name="text">
          <string>Database Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_DatabaseName">
         <property name="text">
          <string>dbname</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_Dummy1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_ExistingConfigurationNames">
         <property name="text">
          <string>Existing Configurations:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTableWidget" name="tableWidget_ExistingConfigurationNames"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_ConfigurationName">
         <property name="text">
          <string>Configuration Name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_ConfigurationName"/>
       </item>
       <item row="5" column="1">
        <widget class="QRadioButton" name="radioButton_tabs">
         <property name="text">
          <string>Tabbed windows</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QRadioButton" name="radioButton_dock">
         <property name="text">
          <string>Docked windows</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
